src/main/java/org/terasology/anotherWorld/generation/PerlinSurfaceHeightProvider.java

```diff
@@ -96,7 +96,7 @@ private float getNoiseInWorld(int worldX, int worldZ) {
                 int zScan = (int) Math.sqrt(scanArea * scanArea - (x - worldX) * (x - worldX));
                 for (int z = worldZ - zScan; z <= worldZ + zScan; z++) {
                     if (z % sampleReduction == 0) {
-                        noiseValue += noise.noise(terrainNoiseMultiplier * x, terrainNoiseMultiplier * z) / noiseScale;
+                        noiseValue += noise.noise(terrainNoiseMultiplier * x, terrainNoiseMultiplier * z) * noiseScale;
```

Comment:
I guess you're reasonably sure that this was a mistake before?

Reply:
its already divided in the default Browniant Noise